pkg/config/config.go

```diff
@@ -48,6 +50,7 @@ wsrep_provider=/usr/lib/galera/libgalera_smm.so
 wsrep_cluster_address="{{ .ClusterAddress }}"
 wsrep_cluster_name=mariadb-operator
 wsrep_slave_threads={{ .Threads }}
+wsrep_provider_options="gmcast.listen_addr=tcp://[::]:4567; ist.recv_addr={{ .Pod }}.{{ .Service }}:4568"
```

Comment:
> ist.recv_addr={{ .Pod }}.{{ .Service }}:4568

why do we need this option to enable IPv6? Isn't  it set to the default value?:
- https://mariadb.com/kb/en/wsrep_provider_options/#istrecv_addr

If that was the case, could we remove it?

> gmcast.listen_addr=tcp://[::]:4567

could we set this conditionally based on whether we have a IPv6 address or not?